ghost/core/core/server/services/tags-public/service.js

```diff
@@ -0,0 +1,31 @@
+class TagsPublicServiceWrapper {
+    async init() {
+        if (this.api) {
+            // Already done
+            return;
+        }
+
+        // Wire up all the dependencies
+        const models = require('../../models');
+        const adapterManager = require('../adapter-manager');
+        const config = require('../../../shared/config');
+
+        let tagsCache;
+        if (config.get('hostSettings:tagsPublicCache:enabled')) {
```

Comment:
Only initializes the cache for tags in hosted environment with a dedicated cache mechanism.